Xamarin.Forms.Core/Items/CarouselView.cs

```diff
@@ -165,6 +165,9 @@ public LinearItemsLayout ItemsLayout
 			set => SetValue(ItemsLayoutProperty, value);
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public bool IsScrolling { get; set; }
```

Comment:
Does `IsScrolling` ever have a different value than `IsDragging`? I'm wondering if we need two separate values, or if we could just have one.

Reply:
Yes, the main difference is:

- IsScrolling: Indicates when the CarouselView is scrolling independently if it is started from ScrollTo  method (programmatically) or from a user interaction.
- IsDragging: Indicates when the user is dragging the Carousel. The Carousel may be scrolling while IsDragging is false.